cli/command/service/list.go

```diff
@@ -2,11 +2,13 @@ package service
 
 import (
 	"context"
-	"fmt"
 	"sort"
 
+	"github.com/docker/docker/client"
 	"vbom.ml/util/sortorder"
 
+	"github.com/docker/docker/api/types/versions"
```

Comment:
need to fix up formatting, but will do so when rebasing after the other PR's are merged